_rules/6cbcbe.md

```diff
@@ -0,0 +1,206 @@
+---
+name: Links with identical accessible names serve equivalent purpose
+test_type: atomic
+
+description: |
+  This rule checks that links with identical accessible names serve an equivalent purpose.
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of link, are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have the same [non-empty](#non-empty) [accessible name](#accessible-name).
```

Comment:
I was wondering if this shouldn't also include links that have a different semantic role. For example, there are several `doc-*` roles that inherit from link. Those are clearly links. Another example of a link with a different role I found is where a an `a[href]` element has `role=menuitem`.

I would say elements with a role that inherit from link should be applicable. I'll leave the menuitem question up to the author.

Reply:
@WilcoFiers, Roles that inherit from the `link` role has now been included. 
For `role=menuitem`, I'd rather not include it in this rule, as the menuitems could be many other things than just links. We might consider writing another rule for it though, like the "iframe has unique name" rule.